### PR TITLE
Added LiquidLava with NPM and new CSS heme

### PR DESCRIPTION
**Pull request by kogarashisan**
Added LiquidLava. with NPM and new CSS theme

### CODE REVIEW

File: examples/lava/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "dependencies": {
+    "lava": "^0.13.2",
+    "director": "~1.2.8",
+    "todomvc-app-css": "=1.0.0",
```
Comment (arthurvr) [inline examples/lava/package.json]: `=` -> `^`

Comment (kogarashisan) [inline examples/lava/package.json]: It's not a bug, it's intended.
This example does not use todomvc-app-css at all, so maybe it's better to remove it completely?

My framework is unable to work with IDs in CSS files, so I have rewritten your index.css into app.css: all ID selectors were renamed into class selectors.

For example: #todoapp -> .todoapp

And one exception:
label[for='toggle-all'] -> .toggle-all-label

P.S. Topic about the framework
https://github.com/tastejs/todomvc/issues/1084

**Comment by sindresorhus** [comment]
@tastejs/todomvc

**Comment by kogarashisan** [comment]
If anyone has quiestions - feel free to ask.

**Comment by samccone** [comment]
Hi @kogarashisan 

Thanks for doing this great work! 
At this time we would like to see your framework mature and grow through use by more users. 
We are open to reevaluating when you feel like you have reached this point, thanks again for the great stuff.

**Pull request closed without merging**